sklearn/metrics/_regression.py

```diff
@@ -601,10 +604,13 @@ def root_mean_squared_error(
         if multioutput == "raw_values":
             return output_errors
         elif multioutput == "uniform_average":
-            # pass None as weights to np.average: uniform mean
+            # pass None as weights to _average: uniform mean
             multioutput = None
 
-    return np.average(output_errors, weights=multioutput)
+    # See comment in mean_absolute_error
```

Comment:
would be nice to have the comment here as well, it's hard to read the code when one needs to go back and forth here.